packages/bpk-component-table/src/BpkTableHead.js

```diff
@@ -19,7 +19,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-const BpkTableHead = props => <thead>{props.children}</thead>;
+const BpkTableHead = props => <thead {...props} />;
```

Comment:
Does passing `children` like this have any caveats or does it just compile to identical code?